packages/remix-react/transition.ts

```diff
@@ -221,6 +221,12 @@ type FetcherStates<TData = any> = {
     submission: ActionSubmission;
     data: TData;
   };
+  LoadingSubmissionActionRedirect: {
+    state: "loading";
+    type: "actionSubmissionRedirect";
```

Comment:
Add call this one `actionRedirect`

So that `fetcher.type` lines up with `transition.type` for the same states.

Reply:
👍  [27e6b23](https://github.com/remix-run/remix/pull/2188/commits/27e6b23fddd3cccd68bb2066b0c412ac75efd3f1)